### PR TITLE
feat(page): add color-gamut support to Page.emulateMediaFeatures

### PR DESCRIPTION
**Pull request by OrKoN**
The change updates the validation function to allow color-gamut media
features and updates the documentation.

Issues: #6761

### CODE REVIEW

**Comment by OrKoN** [comment]
@googlebot I consent.

File: src/common/Page.ts [hunk 1]
```diff
@@ -1387,7 +1387,7 @@ export class Page extends EventEmitter {
       features.every((mediaFeature) => {
         const name = mediaFeature.name;
         assert(
-          /^prefers-(?:color-scheme|reduced-motion)$/.test(name),
+          /^(prefers-(?:color-scheme|reduced-motion)|color-gamut)$/.test(name),
```
Comment (mathiasbynens) [inline src/common/Page.ts]: No need to capture:

```suggestion
          /^(?:prefers-(?:color-scheme|reduced-motion)|color-gamut)$/.test(name),
```

**Review by mathiasbynens — APPROVED** [review]

File: docs/api.md [hunk 2]
```diff
@@ -1440,6 +1440,16 @@ await page.evaluate(() => matchMedia('(prefers-reduced-motion: reduce)').matches
...
+
+await page.emulateMediaFeatures([
+  { name: 'color-gamut', value: 'p3' },
+]);
+await page.evaluate(() => matchMedia('(color-gamu: srgb)').matches);
+// → true
+await page.evaluate(() => matchMedia('(color-gamu: p3)').matches);
+// → true
+await page.evaluate(() => matchMedia('(color-gamu: rec2020)').matches);
+// → false
```
Comment (mathiasbynens) [inline docs/api.md]:
```suggestion
await page.evaluate(() => matchMedia('(color-gamut: srgb)').matches);
// → true
await page.evaluate(() => matchMedia('(color-gamut: p3)').matches);
// → true
await page.evaluate(() => matchMedia('(color-gamut: rec2020)').matches);
// → false
```

File: docs/api.md [hunk 3]
```diff
@@ -1411,7 +1411,7 @@ List of all available devices is available in the source code: [src/common/Devic
 
 #### page.emulateMediaFeatures(features)
 - `features` <?[Array]<[Object]>> Given an array of media feature objects, emulates CSS media features on the page. Each media feature object must have the following properties:
-  - `name` <[string]> The CSS media feature name. Supported names are `'prefers-colors-scheme'` and `'prefers-reduced-motion'`.
+  - `name` <[string]> The CSS media feature name. Supported names are `'prefers-colors-scheme'`, `'prefers-reduced-motion'` and `'color-gamut'`.
```
Comment (mathiasbynens) [inline docs/api.md]: Oxford comma 4 lyfe

```suggestion
  - `name` <[string]> The CSS media feature name. Supported names are `'prefers-colors-scheme'`, `'prefers-reduced-motion'`, and `'color-gamut'`.
```

**Pull request merged**